Syntax error in Keycloak source code in main

### Describe the bug

I'm using JDT / Eclipse Java Compier to compile Keycloak in vscode and I'm getting syntax error on the double semicolons here:

https://github.com/keycloak/keycloak/blob/9b889b44b413e889603bc9411ff60377f96b4f28/model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/InfinispanUserCacheProviderFactory.java#L33-L34

It works with JDK compiler but not JDT. I found explanation here https://stackoverflow.com/questions/8125558/eclipse-double-semi-colon-on-an-import.

### Version

main

### Expected behavior

_No response_

### Actual behavior

```
Caused by: java.lang.Error: Unresolved compilation problem: 
	Syntax error on token ";", delete this token
```

### How to Reproduce?

_No response_

### Anything else?

_No response_

--- a/model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/InfinispanUserCacheProviderFactory.java
+++ b/model/infinispan/src/main/java/org/keycloak/models/cache/infinispan/InfinispanUserCacheProviderFactory.java
@@ -30,8 +30,6 @@ import org.keycloak.models.cache.UserCacheProviderFactory;
 import org.keycloak.models.cache.infinispan.entities.Revisioned;
 import org.keycloak.models.cache.infinispan.events.InvalidationEvent;
 import org.keycloak.provider.InvalidationHandler;
-import org.keycloak.provider.InvalidationHandler.InvalidableObjectType;;
-import org.keycloak.provider.InvalidationHandler.ObjectType;;
 
 /**
  * @author <a href="mailto:sthorger@redhat.com">Stian Thorgersen</a>

